src/shared/mount-util.c

```diff
@@ -1087,24 +1087,29 @@ static int mount_in_namespace(
         pid_t child;
         int r;
 
-        assert(target > 0);
         assert(propagate_path);
         assert(incoming_path);
         assert(src);
         assert(dest);
         assert(!options || is_image);
 
-        r = namespace_open(target, &pidns_fd, &mntns_fd, NULL, NULL, &root_fd);
+        if (!pidref_is_set(target))
```

Comment:
Hmm, maybe keep this as an assertion?

Reply:
Other uses of PidRef never assert, just run this same check